openvmm/openvmm_entry/src/cli_args.rs

```diff
@@ -626,6 +626,17 @@ pub enum DiskCliKind {
     Memory(u64),
     // memdiff:<kind>
     MemoryDiff(Box<DiskCliKind>),
+    // sql:<path>[;create=<len>]
```

Comment:
Having two things is fine, but maybe we should come up with a way to make arbitrary layered disks to support all the crazy combinations.

Or should we just build the separate manifest thing?

Reply:
I think its reasonable to have "shortcuts" in the dev-facing UI, even with a more robust and flexible alternative command in place. e.g: I know QEMU has many CLI flags that decompose into a set of more verbose, but semantically equivalent CLI flags.